style.css

```diff
@@ -0,0 +1,100 @@
+@import 'https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap';
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.container {
+  display: grid;
+  width: fit-content;
+  height: fit-content;
+  background: #e5e5e5;
+}
+
+body {
+  width: 100%;
+  background: #f1f1f1;
+}
+
+.main-header {
+  height: 50px;
+}
+
+.menu {
+  position: absolute;
+  left: 80%;
+  top: 4%;
+  font-size: 24px;
+}
+
+.main-header .menu {
+  left: 30;
+  line-height: 4;
+  display: flex;
+}
+
+.part1 {
+  border: 1px solid rgb(202, 199, 199);
```

Comment:
- [ ] Kindly remove the border of the headline section to follow the Figma design as requested
### Your design :point_down: 
![Screenshot from 2023-05-29 09-31-35](https://github.com/mumbereh/my-portfolio/assets/91142432/22231913-0f0b-44cc-b579-cab919f1bfd8)
### what it is supposed to look like :point_down: 
![Screenshot from 2023-05-29 09-31-09](https://github.com/mumbereh/my-portfolio/assets/91142432/3746c423-b639-431d-989e-def5d4059ecd)